compiler/ilgen/IlBuilder.hpp

```diff
@@ -1,14 +1,12 @@
 /*******************************************************************************
- * Copyright (c) 2016, 2018 IBM Corp. and others
+ * Copyright (c) 2016, 2022 IBM Corp. and others
  *
  * This program and the accompanying materials are made available under
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at http://eclipse.org/legal/epl-2.0
  * or the Apache License, Version 2.0 which accompanies this distribution
  * and is available at https://www.apache.org/licenses/LICENSE-2.0.
  *
- * This Source Code may also be made available under the following Secondary
- * Licenses when the conditions for such availability set forth in the
```

Comment:
Was the removal of this part of the license intentional?

Reply:
uh, no. good catch again!

Reply:
Pushed an update